compiler/rustc_data_structures/src/marker.rs

```diff
@@ -147,14 +147,14 @@ cfg_match! {
             [crate::owned_slice::OwnedSlice]
         );
 
-        // PowerPC and MIPS platforms with 32-bit pointers do not
+        // MIPS, PowerPC and SPARC platforms with 32-bit pointers do not
         // have AtomicU64 type.
-        #[cfg(not(any(target_arch = "powerpc", target_arch = "mips")))]
+        #[cfg(not(any(target_arch = "powerpc", target_arch = "powerpc", target_arch = "sparc")))]
```

Comment:
You seem to have listed powerpc twice

Reply:
Oh shoot, you're right. I will send a follow-up PR to fix this. Thanks a lot for catching this!

Reply:
Created https://github.com/rust-lang/rust/pull/127075 now. Again, thanks so much for catching this!